### ISSUE TITLE
There is some problem in supporting continuously training

### ISSUE THREAD

**Issue by ZacharyWaseda** [huggingface/pytorch-pretrained-BERT#81]
I change the run_classfifier.py in order to support continuously training. i save the model.state_dict() and the BertAdam optimizer.state_dict(), and I load them when start continuously training. However, After some epochs, the loss will increase little by little and finally end with a large loss value. I do not know the reason. Please help me.

**Comment by thomwolf** [comment]
Hi @ZacharyWaseda, continuous training is an open-research problem. You should rather seek some solution in the papers/workshop/conference discussing researches in this field. This is not my personal field of expertise so I can only direct you to google and other search engine for more information.
